crates/rolldown_binding/src/options/plugin/plugin.rs

```diff
@@ -172,3 +204,220 @@ impl From<rolldown_common::RenderedChunk> for RenderedChunk {
     }
   }
 }
+
+impl PluginOptions {
```

Comment:
A new-type wrapped `PluginOptions` would be more flexable. We might add custom data into the js plugin in rust side for debugging or other purposes. Using `PluginOptions` directly would lose this kind of adavantage.